libsplinter/src/admin/service/messages/mod.rs

```diff
@@ -210,6 +210,75 @@ impl CreateCircuit {
 
         Ok(create_request)
     }
+
+    pub fn into_circuit_proto(self) -> Result<admin::Circuit, MarshallingError> {
```

Comment:
This seems a bit odd to include here, in a struct that transforms to several different types. Perhaps in a future PR, this should be moved to a `TryInto` implementation, or some other strategy.

Reply:
I added these comments (besides the separate commit one; I addressed in the PR) to the `circuit-disband` feature doc so these will get revisited in the stabilization process